NobankCore/noBank/src/main/java/com/hunk/nobank/util/ViewHelper.java

```diff
@@ -210,8 +210,8 @@ public static void updateFontsStyle(ViewGroup vg) {
 
     private static void setFontByConfig(TextView tv) {
         TypefaceCache cache = Core.getInstance().getTypefaceCache();
-        if (tv.getTypeface() == null) {
-            Logging.d("setFontByConfig meet a null pointer problem and this text view id = " + tv.getId());
+        if (tv.getTypeface() == null) { // default font won't have typeface.
+            tv.setTypeface(cache.NORMAL);
```

Comment:
Default setting will be null when call getTypeface(). I didn't understand that before.